.changeset/wild-dots-bake.md

```diff
@@ -0,0 +1,20 @@
+---
+"@internal/check-imports": patch
+"@fuel-ts/transactions": patch
+"@fuel-ts/abi-typegen": patch
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/account": patch
+"@fuel-ts/address": patch
+"@fuel-ts/program": patch
+"@fuel-ts/recipes": patch
+"@fuel-ts/crypto": patch
+"@fuel-ts/errors": patch
+"@fuel-ts/hasher": patch
+"@fuel-ts/logger": patch
+"@fuel-ts/script": patch
+"fuels": minor
+"@fuel-ts/utils": patch
+---
+
+chore!: redistributed the `@fuel-ts/interfaces` package
```

Comment:
@nedsalk @Torres-ssf Do you remember why we couldn't do this in the past? I think it was some cyclic dependencies or something, and I wonder if we're missing something here.

Reply:
I believe that there were some cyclic dependencies with regards to accounts and providers but they might have been resolved with the merger of those packages into the single `@fuel-ts/account` package. I feel like the removal of `AbstractProgram` should've also caused some issues, but perhaps we have refactored out the logic which depended on its existence. 

All in all, if the CI is green then IMO we can proceed reviewing and ultimately merging it.